learn/react-native/pdf-preview-and-download.md

```diff
@@ -0,0 +1,73 @@
+---
+title: "PDF preview"
+id: "pdf-preview-and-download"
+sidebar_label: "PDF preview"
+---
+---
+import PdfPreview from '/learn/assets/pdf-preview-and-download.png';
+
+Viewing PDFs without leaving the mobile app offers ability for developers to keep their users in the app's context. It is possible to let PDF file download to the phone and have the user open it using a different app. That may not be best user experience for an app. Not all users may know how to locate and open the PDF file in a different app. 
+In this blog we will see how to add PDF preview functionality to a WaveMaker app using the `react-native-pdf` and 
```

Comment:
In this 'how-to' or In this article

Reply:
@prasanth-reddy-wavemaker-com react-native-pdf supports password protected pdf file. Also, We will check if the preview works for the password protected PDF file or not.

![Screenshot 2023-09-29 163604](https://github.com/wavemaker/docs/assets/115488594/4ecbb718-8b5d-4831-8c97-5cbf77494ca7)